### PR TITLE
feat(webkit): roll to r1555

### PR DESCRIPTION
**Pull request by mxschmitt**
Supersedes https://github.com/microsoft/playwright/pull/9266

### CODE REVIEW

File: tests/page/page-request-fulfill.spec.ts [hunk 1]
```diff
@@ -286,7 +284,6 @@ it('should fulfill with fetch response that has multiple set-cookie', async ({ p
 });
 
 it('headerValue should return set-cookie from intercepted response', async ({ page, server, browserName }) => {
-  it.fail(browserName === 'chromium', 'Set-Cookie is missing in response after interception');
```
Comment (yury-s) [inline tests/page/page-request-fulfill.spec.ts:289]: Revert this.

**Review by yury-s — COMMENTED** [review]
We are still waiting for arm build.

**Comment by mxschmitt** [comment]
> We are still waiting for arm build.

Yeah was more to check if last WebKit roll was a healthy one, so we can unblock dpino, but seems like wheel on ubuntu is broken, let's wait again for the bots.

**Comment by yury-s** [comment]
Closing in favor of #9310

**Pull request closed without merging**